### PR TITLE
Nfdiv 722 citizen confirm joint app

### PR DESCRIPTION
**Pull request by lara-mpt**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/NFDIV-722

### Change description ###
Create prayerHasBeenGiven and statementOfTruth fields for app2.
Create event and state Applicant2Approve

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/citizen/event/Applicant2Approve.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    @Override
+    public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+
+        configBuilder
+            .event(APPLICANT_2_APPROVE)
+            .forStateTransition(AwaitingApplicant2Response, Applicant2Approved)
+            .name("Applicant 2 approve")
+            .description("Applicant 2 has approved")
+            .grant(CREATE_READ_UPDATE, APPLICANT_2)
+            .displayOrder(1)
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/divorce/citizen/event/Applicant2Approve.java:36]: I think we should omit this now

**Review by linusnorton — APPROVED** [review]
I think it's fine for now but we should consider making a class for the applicant1 and applicant2 questions in here

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/State.java [hunk 2]
```diff
@@ -53,6 +54,20 @@
     )
     AosOverdue("AosOverdue"),
 
+    @CCD(
+        name = "Applicant 2 approved",
+        label = "# **${[CASE_REFERENCE]}** ${applicant1LastName} **&** ${applicant2LastName}\n### **${[STATE]}**\n",
+        access = {CaseAccessAdministrator.class}
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/divorcecase/model/State.java:60]: @linusnorton do we need CAA access on this one ?

File: src/test/java/uk/gov/hmcts/divorce/citizen/event/Applicant2ApproveTest.java [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    @InjectMocks
+    private Applicant2Approve applicant2Approve;
+
+    @Test
+    void shouldAddConfigurationToConfigBuilder() {
+        final ConfigBuilderImpl<CaseData, State, UserRole> configBuilder = createCaseDataConfigBuilder();
+
+        applicant2Approve.configure(configBuilder);
+
+        Assertions.assertThat(getEventsFrom(configBuilder).values())
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/divorce/citizen/event/Applicant2ApproveTest.java:46]: can static import Assertions

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/State.java (hunk 2 above)
Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/divorce/divorcecase/model/State.java:60]: no, good catch

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**